src/typesetting/typeset.ts

```diff
@@ -42,7 +42,7 @@ export function embed(type: EmbedType) {
 }
 
 export type FromDom = (node: Node) => any;
-export type LineData = [attributes: AttributeMap, children: VChild[] ];
+export type LineData = [attributes: AttributeMap, children: VChild[], id:string];
```

Comment:
`renderMultiple` needs the `id` to set the key for line elements, and since `id` is no longer at `attributes`, it must be passed in separately.